.github/workflows/close-issue.yml

```diff
@@ -0,0 +1,22 @@
+name: Close inactive issues
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
On some systems, having cronjob `0 0 * * *` (midnight UTC) may cause weird issues because everyone have the same cronjob that run at the same time (which cause overload), for example [let's encrypt](https://community.letsencrypt.org/t/brief-overloads-at-midnight-utc-a-request-for-help/179993) recommends to run at 0h02, 0h03,... instead of 0h00.